app/build.gradle

```diff
@@ -1,7 +1,6 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
-    id 'com.getkeepsafe.dexcount'
```

Comment:
fun fact was taking up 30% of compilation time

Reply:
:face_with_head_bandage: